docs: Helm Charts 5.2.0 release notes (#1824)
TykTechnologies/tyk-docs: developer-support/release-notes/helm-chart.mdx

---

## 5.2 Release Notes

### 5.2.0 Release Notes

#### Release Date xx April 2026

#### Release Highlights
Tyk Charts 5.2.0 enhances Helm chart customization capabilities by adding pod labeling support for the Tyk Operator.

> 

Reviewer: ```suggestion

This version aligns with Tyk LTS release 5.8.14.

```

Given the time difference between your upgrade and the release of this version, we recommend customers verify the ongoing support of third-party dependencies they install, as their status may have changed since the release.

#### Deprecations
There are no deprecations in this release.

#### Upgrade instructions
You can use helm upgrade to upgrade your release
```bash
helm repo add tyk-helm https://helm.tyk.io/public/helm/charts/
helm repo update
helm upgrade [RELEASE_NAME] tyk-helm/[CHART_NAME]
```

#### Downloads
- [Source code](https://github.com/TykTechnologies/tyk-charts/archive/refs/tags/v5.2.0.tar.gz)
- [ArtifactHub - tyk-stack](https://artifacthub.io/packages/helm/tyk-helm/tyk-stack/5.2.0)
- [ArtifactHub - tyk-control-plane](https://artifacthub.io/packages/helm/tyk-helm/tyk-control-plane/5.2.0)
- [ArtifactHub - tyk-data-plane](https://artifacthub.io/packages/helm/tyk-helm/tyk-data-plane/5.2.0)
- [ArtifactHub - tyk-oss](https://artifacthub.io/packages/helm/tyk-helm/tyk-oss/5.2.0)

#### Changelog
<a id="Changelog-v5.2.0" data-scroll-offset></a>

##### Added

> <Accordion title='Add `podLabels` support to Tyk Operator in helm chart'>

Reviewer: ```suggestion
<Accordion title='Add `podLabels` support for Tyk Operator'>
```

> Added support for specifying custom `podLabels` for the Tyk Operator deployment in Helm charts. This allows users to attach custom labels to the underlying Tyk Operator pods, bringing it in line with the other Tyk components.

Reviewer: ```suggestion
Added support for users to attach custom labels to Tyk Operator pods using the `podLabels` map. This functionality is already available for the Tyk Dashboard, Tyk Gateway, Tyk Developer Portal and Tyk Pump components.
```

---

## 5.2 Release Notes

### 5.2.0 Release Notes

> #### Release Date 08 May 2026

Author: Needs to be updated

> #### Release Date 08 May 2026

Reviewer 2: ```suggestion
#### Release Date 21 May 2026
```
